### ISSUE TITLE
missing dependencies (user banned?)

### ISSUE THREAD

**Issue by despair86** [dutchcoders/transfer.sh#281]
```
bucket@bucket:~$ go get github.com/dutchcoders/transfer.sh
package github.com/russross/blackfriday/v2: cannot find package "github.com/russross/blackfriday/v2" in any of:
        /opt/local/go112/src/github.com/russross/blackfriday/v2 (from $GOROOT)
        /home/bucket/go/src/github.com/russross/blackfriday/v2 (from $GOPATH)
```

**Comment by aspacca** [comment]
@despair86 transfer.sh uses gomodule, can you try with `GO111MODULE=on`?
also, what version of go are you using?

**Comment by despair86** [comment]
oh I'm on go 1.12-13, but the repo itself no longer exists regardless

**Comment by aspacca** [comment]
the repo exists: https://github.com/russross/blackfriday

**Comment by aspacca** [comment]
https://github.com/golang/go/issues/36391

I've opened this: in the meanwhile you can fix with cloning the repo and doing a `go get ./...`

**Comment by aspacca** [comment]
@despair86 fixed in v1.1.3
